src/lib.rs

```diff
@@ -30,6 +30,9 @@ use raw_window_handle::{
     HasRawDisplayHandle, HasRawWindowHandle, RawDisplayHandle, RawWindowHandle,
 };
 
+#[cfg(target_arch = "wasm32")]
+pub use self::web::SurfaceExtWeb;
```

Comment:
Wasn't sure where to actually define and implement the trait, I hope this is alright.

Reply:
This works for me!